C7/UIElements/GameStatus/GameStatus.cs

```diff
@@ -33,7 +38,7 @@ private void OnTurnEnded()
 	private void OnTurnStarted(int turnNumber)
 	{		
 		//Oh hai, we do need this handler here!
-		LowerRightInfoBox.SetTurn(turnNumber);
+		//LowerRightInfoBox.SetTurn(turnNumber);
```

Comment:
Probably can take this method out now, I think... or maybe we should leave it for now since there are other things that will need updated as we add more features?